core/shared/src/main/scala/scalaz/zio/FiberFailure.scala

```diff
@@ -78,11 +78,15 @@ final case class FiberFailure(cause: Cause[Any]) extends Throwable {
     def causeToSequential(cause: Cause[Any]): Sequential =
       cause match {
         case Cause.Fail(t: Throwable) =>
-          Sequential(List(Failure(List("A checked error was not handled: ") ++ t.getStackTrace.map(_.toString))))
+          Sequential(
+            List(Failure(List(s"A checked error was not handled: ${t.getMessage}") ++ t.getStackTrace.map(_.toString)))
```

Comment:
Would be it be possible to correctly indent this? (i.e. reformat the otherwise fully left-indented stack trace)

Reply:
Sure. How exactly should this be formatted?
Currently an Exception would be reported like this:

scalaz.zio.FiberFailure: Fiber failed.
╥
╠─An unchecked error was produced: foo
║  scalaz.zio.testkit.SchedulerSpec.$anonfun$e3$11(SchedulerSpec.scala:64)
║  scalaz.zio.testkit.SchedulerSpec.$anonfun$e3$11$adapted(SchedulerSpec.scala:63)
║  scalaz.zio.internal.FiberContext.nextInstr(FiberContext.scala:429)
║  scalaz.zio.internal.FiberContext.evaluateNow(FiberContext.scala:176)
║  scalaz.zio.internal.FiberContext.$anonfun$evaluateLater$1(FiberContext.scala:338)
║  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
║  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
║  java.lang.Thread.run(Thread.java:748)
▼

How should it instead be displayed?

Another thing I thought about that this 'arrow' reposting style does not really make sense for simple failures. Maybe something like this?
```
    ("Fiber failed." :: { sequence match {
      // use simple report for single failures
      case Sequential(List(Failure(cause))) => cause

      case _ => format(sequence).updated(0, "╥")
    }}).mkString("\n")
```

Reply:
Ah, I didn't know it would be formatted like that already (I thought it would not print with the arrow). 

You're right for simple failures, a simpler option could be used.

Reply:
I updated how throwables are rendered so they look the same way as when
they are normally thrown. Also exit causes with a single underlying reason
will now be directly reported (no arrows will be drawn).

```
scalaz.zio.FiberFailure: Fiber failed.
╥
╠─An unchecked error was produced.
║ java.lang.RuntimeException: foo
║       at scalaz.zio.testkit.RandomSpec.$anonfun$nextIntWithLimit$5(RandomSpec.scala:70)
║       at scalaz.zio.testkit.RandomSpec.$anonfun$nextIntWithLimit$5$adapted(RandomSpec.scala:69)
║       at scalaz.zio.internal.FiberContext.nextInstr(FiberContext.scala:429)
║       at scalaz.zio.internal.FiberContext.evaluateNow(FiberContext.scala:176)
║       at scalaz.zio.Runtime.unsafeRunAsync(Runtime.scala:70)
║       at scalaz.zio.Runtime.unsafeRunAsync$(Runtime.scala:67)
║       at scalaz.zio.TestRuntime.unsafeRunAsync(TestRuntime.scala:10)
║       at scalaz.zio.Runtime.unsafeRunSync(Runtime.scala:56)
║       at scalaz.zio.Runtime.unsafeRunSync$(Runtime.scala:53)
║       at scalaz.zio.TestRuntime.unsafeRunSync(TestRuntime.scala:10)
║       at scalaz.zio.Runtime.unsafeRun(Runtime.scala:45)
...
```
```
scalaz.zio.FiberFailure: Fiber failed.
An unchecked error was produced.
java.lang.RuntimeException: foo
        at scalaz.zio.testkit.SchedulerSpec.$anonfun$e3$11(SchedulerSpec.scala:64)
        at scalaz.zio.testkit.SchedulerSpec.$anonfun$e3$11$adapted(SchedulerSpec.scala:63)
        at scalaz.zio.internal.FiberContext.nextInstr(FiberContext.scala:429)
        at scalaz.zio.internal.FiberContext.evaluateNow(FiberContext.scala:176)
        at scalaz.zio.internal.FiberContext.$anonfun$evaluateLater$1(FiberContext.scala:338)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
        at scalaz.zio.Runtime.$anonfun$unsafeRun$1(Runtime.scala:45)
        at scalaz.zio.Exit.getOrElse(Exit.scala:31)
        at scalaz.zio.Exit.getOrElse$(Exit.scala:29)
```

One thing that is still quite ugly is how the arrow style reporting plays together with the stacktrace
of the fiberfailure itself, but I don't have any ideas there currently.

Reply:
@mschuwalow 

Here are some ideas on that:

1. First, disable stack traces on `FiberFailure`, since we don't need them.
2. Second, override `printStackTrace`.